app/models/buyer_domains.py

```diff
@@ -0,0 +1,14 @@
+from app import db
+
+
+class BuyerEmailDomain(db.Model):
+    __tablename__ = 'buyer_email_domains'
+
+    id = db.Column(db.Integer, primary_key=True)
+    domain_name = db.Column(db.String(255), nullable=False)
```

Comment:
We dropped the 255 from the table definition, should also drop here.

Reply:
Yep - that should disappear on the rebase.

Reply:
OK, sorry, haven't figured out what belongs to what.